boot/boot_serial/src/boot_serial.c

```diff
@@ -58,6 +58,12 @@
 #include "bootutil_priv.h"
 #endif
 
+#ifdef CONFIG_ENABLE_MGMT_PERUSER
```

Comment:
I would use mediation of mcuboot_config.h file

Reply:
OK, will do.